### DISCUSSION TITLE
[Feature request] [Playlists] Add parent folder in dashboard selection

### DISCUSSION THREAD

**Discussion by BasvanH** [grafana/grafana#37674] [General]
Hello,

When creating/editting a playlist, you see a list of availeble dashboards. We have same dashboard names in different folders for different purposses. This makes it hard to select the correct dashboard. We are using tags for this, but still it would help if the parent folder is visually added to the view.

So it will look like something like this:
![image](https://user-images.githubusercontent.com/1708136/128669780-c269e492-e216-461d-be9f-a421670916fe.png)

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
